Source/Scene/Scene.js

```diff
@@ -176,6 +176,59 @@ define([
         };
     };
 
+    function View(scene, camera, viewport) {
```

Comment:
Move this to it's own file. I could see this doing a lot more than just being a container. For example, having it's own camera. Creating it's own potentially visible set.